bindings/rust/s2n-tls/src/testing/s2n_tls.rs

```diff
@@ -1030,4 +1030,14 @@ mod tests {
             .unwrap();
         assert_eq!(context.invoked_count, 1);
     }
+
+    #[test]
+    fn client_hello_version() -> Result<(), testing::Error> {
```

Comment:
Probably need to update this test name.

It would also be nice to have a "true" case here, since the comparison lives in the Rust instead of the underlying C. Although I'm not sure that's feasible / worth it for these tests. In our unit tests we generally use a hardcoded sslv3 hello for testing, which could be messy here. Maybe you could modify a non-sslv2 client hello to look like an sslv3 client hello?

Reply:
I think the hard-coded approach ended up being pretty readable, let me know if you want me to try the other approach though.